.babelrc

```diff
@@ -1,14 +0,0 @@
-{
-  "plugins": [
-    "version"
-  ],
-  "env": {
-    "production": {
-      "plugins": [
-        "babel-plugin-jsx-remove-data-test-id",
-        "@babel/proposal-nullish-coalescing-operator",
```

Comment:
The file .babelrc was renamed to babel.config.js. The plugins `@babel/proposal-nullish-coalescing-operator` and `@babel/proposal-optional-chaining` are no longer maintained since the proposal was accepted in ES2020. Now there are 2 new plugins that I added in babel.config.js: `@babel/plugin-transform-nullish-coalescing-operator` and `@babel/plugin-transform-optional-chaining`.